module/zcommon/simd_stat.c

```diff
@@ -0,0 +1,167 @@
+#include <sys/zfs_context.h>
+#include <sys/kstat.h>
+#include <sys/simd.h>
+
+
+#ifdef _KERNEL
+#ifdef __linux__
+#include <linux/simd.h>
+#endif /* __linux__ */
+kstat_t *simd_stat_kstat;
+#endif /* _KERNEL */
+
+#ifdef _KERNEL
+/* shut up making this more lines helps nobody */
```

Comment:
Appeasing the style checker and dropping both comments actually saves you a line.

```c
#define SIMD_STAT_PRINT(s, feat, val) \
    kmem_scnprintf(s + off, 4095 - off, "%-16s\t%1d\n", feat, (val))
```